packages/html/src/index.ts

```diff
@@ -1,20 +1,36 @@
-const { extname } = require('path');
+import { extname } from 'path';
 
-const getFiles = (bundle) => {
+import { Plugin, NormalizedOutputOptions, OutputBundle, EmittedAsset } from 'rollup';
+
+import { RollupPluginHtmlOptions, RollupPluginHtmlTemplateOptions } from '../types';
+
+const getFiles = (bundle: OutputBundle): RollupPluginHtmlTemplateOptions['files'] => {
   const files = Object.values(bundle).filter(
-    (file) => file.isEntry || (typeof file.type === 'string' ? file.type === 'asset' : file.isAsset)
+    (file) =>
+      file.type === 'chunk' ||
+      (typeof file.type === 'string' ? file.type === 'asset' : file.isAsset)
   );
-  const result = {};
+  const result = {} as ReturnType<typeof getFiles>;
   for (const file of files) {
     const { fileName } = file;
     const extension = extname(fileName).substring(1);
-    result[extension] = (result[extension] || []).concat(file);
+
+    switch (extension) {
+      case 'css':
+      case 'js':
+      case 'map':
+        result[extension] = (result[extension] || []).concat(file);
+        break;
+      default:
+        // TODO: verify if this branch can be reached or not.
```

Comment:
I wanted to ask a question here because I don't have the rollup knowledge to answer this one. The js code here just grabbed the file extension and put it on the result object without checking what it actually is. Can it happen that rollup emits files that are not `js`, `css` or `map`? For example `wasm` or an `svg` or something?

Because if it can than my question is do we want those on the `files` object and if yes, what type of files can we expect here so I can expand the typing and change the implementation so they actually end up on the `files` object.

Reply:
It's a good question - but not one to be tackled during a refactor. Let's focus on parity here and we can open a new issue to discuss how this should be handled moving forward. The intent behind this plugin is to setup some very basic default templating, and anything beyond that, empower the user to manipulate what they need for a more custom or nuanced solution. We don't want to be in the business of anticipating a user's every move. That's how things get to an unmaintainable state.

Reply:
You're right, I have a PR open that aims to update the docs around this, I think I'll throw together a rollup config that let's me import `svg`s `jpg`s and various file formats, see what ends up in the `files` object and then update that PR with the appropriate documentation changes.

In the meantime I've reverted this back to its original form.